### ISSUE TITLE
Cannot expand Sets in state viewer

### ISSUE THREAD

**Issue by mnpenner** [facebook/react-devtools#435]
When I click the little arrow beside a [Set](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set) in the state viewer, it does not expand like with other objects and arrays.

I've created a [fiddle](https://jsfiddle.net/mnpenner/xwx1dhep/), but I can't figure out how to open React Dev Tools there. Here's a [raw gist](https://cdn.rawgit.com/mnpenner/6ea63bd3e15473b72a71513a49ba794f/raw/9678dad7e19442519c594e8c30d290f9cd7c641f/react-devtools-set.html) that demonstrates the problem.

``` js
var Hello = React.createClass({
  getInitialState: function() {
    return {items: new Set([1,2,3])};
  },

  render: function() {
    return <div>{'{' + Array.from(this.state.items).join(',') + '}'}</div>;
  }
});

ReactDOM.render(
  <Hello name="World" />,
  document.getElementById('container')
);
```

![image](https://cloud.githubusercontent.com/assets/1112998/18599370/06b0dd5e-7c0d-11e6-97db-fd158404743b.png)

**Comment by G1enY0ung** [comment]
+1, same situation

**Comment by gaearon** [comment]
This will be fixed by https://github.com/facebook/react-devtools/pull/459 if it gets merged.

**Comment by bright-star** [comment]
Another way this error may log is as:

```
Uncaught TypeError: Cannot read property 'size' of undefined
    at Set.get 
```

**Comment by gaearon** [comment]
Seems like this is still an issue after #459.

**Comment by gaearon** [comment]
Hmm, I think I made a mistake while testing. It does seem to work on master for me now.
So closed by #459, will release the fix soon.
